test/plugins/test_lastgenre.py

```diff
@@ -409,12 +411,13 @@ def test_sort_by_depth(self):
                 "separator": "\u0000",
                 "canonical": False,
                 "prefer_specific": False,
+                "count": 10,
```

Comment:
Why?

And why are these test cases left in place, i.e. the ones that test the separator which has no effect any more?